packages/cascara/src/ui/Table/TableHeader.js

```diff
@@ -5,14 +5,17 @@ import ErrorBoundary from '../../shared/ErrorBoundary';
 import { ModuleContext } from '../../modules/context';
 
 const TableHeader = () => {
-  const { dataConfig = {} } = useContext(ModuleContext);
+  // FDS-164: table header not adding an extra column for actions
+  // when new prop actions is passed.
```

Comment:
@lordpixel can we please add a test for this? It's definitely something that is easy to miss (as we did). This can be created as a separate task, but let's definitely get it in.